### DISCUSSION TITLE
Sec Onion and Elastic interaction

### DISCUSSION THREAD

**Discussion by TheDude2741** [Security-Onion-Solutions/securityonion#12789] [General] [answered]
Hello, 
There are some features within Elastic that are not showing even though we have a license. Does Sec Onion limi Elastic features by chance?

**Comment by cm-ops** [comment, marked as answer]
There are some features disabled by default. https://docs.securityonion.net/en/2.4/kibana.html#features `Kibana > Stack Management > Spaces > Default` then turn on what you want.

**Reply by TheDude2741** [reply]
Worked like a charm. Very much appreciated!

**Reply by TheDude2741** [reply]
Worked like a charm. Very much appreciated!
